src/components/messenger/list/index.tsx

```diff
@@ -155,7 +155,16 @@ export class Container extends React.Component<Properties, State> {
     return users.map((user) => ({ ...user, image: user.profileImage }));
   };
 
+  isUserAlreadyInConversation = (userId: string) => {
+    return this.props.conversations.find((c) => c.otherMembers[0].userId === userId);
```

Comment:
Does this make sense? What if the user is in multiple group conversations and just happens to be the first one listed?

Reply:
yes just thought about that before seeing your comment 😅 , i'll filter for only "one2one" convo's

Reply:
updated